### DISCUSSION TITLE
Unicode @ suggestion

### DISCUSSION THREAD

**Discussion by rstanuwijaya** [James-Yu/LaTeX-Workshop#3508] [Q&A]
I'm thinking of adding an option to use unicode characters for `@` sugesstions, like [this](https://github.com/rstanuwijaya/LaTeX-Workshop/blob/unicode-replace/data/at-suggestions-unicode.json). 

I will figure out how to add VS code setting toggle later.

Any feedback?

**Comment by jlelong** [comment]
You can redefine the default @-suggestions by using [`latex-workshop.intellisense.atSuggestionJSON.replace`](https://github.com/James-Yu/LaTeX-Workshop/wiki/Intellisense#latex-workshopintellisenseatsuggestionjsonreplace).
